gallery/src/pageConfig.ts

```diff
@@ -20,7 +20,6 @@ import NxLoadWrapperPage from './components/NxLoadWrapper/NxLoadWrapperPage';
 import NxModalPage from './components/NxModal/NxModalPage';
 import NxPageTitlePage from './styles/NxPageTitle/NxPageTitlePage';
 import NxTilePage from './styles/NxTile/NxTilePage';
-import NxBtnPage from './styles/NxBtn/NxBtnPage';
```

Comment:
Shouldn't this be in the HTML variants section?

Reply:
> 

https://github.com/sonatype/sonatype-react-shared-components/pull/596/commits/a232f94f97b553b973d7da0cd835b3ee3d8c958c